tests/runtime/filter_wasm.c

```diff
@@ -146,6 +153,9 @@ void flb_test_append_tag(void)
     char *result;
     struct flb_lib_out_cb cb_data;
 
+    /* clear previous output */
+    clear_output();
```

Comment:
This is to remove following valgrind error.
It reports the pointer is already freed and it means the test uses previous output.

```
Test numeric_records...                         ==29372== Invalid read of size 1
==29372==    at 0x4853A6E: strstr (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==29372==    by 0x1483B9: flb_test_numerics_records (filter_wasm.c:282)
==29372==    by 0x145DDD: test_do_run_ (acutest.h:1007)
==29372==    by 0x146208: test_run_ (acutest.h:1178)
==29372==    by 0x14745B: main (acutest.h:1700)
==29372==  Address 0x72d1ba0 is 0 bytes inside a block of size 63 free'd
==29372==    at 0x484B27F: free (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==29372==    by 0x149BB8: flb_free (flb_mem.h:127)
==29372==    by 0x14E849: flb_lib_free (flb_lib.c:591)
==29372==    by 0x147E84: flb_test_append_tag (filter_wasm.c:188)
==29372==    by 0x145DDD: test_do_run_ (acutest.h:1007)
==29372==    by 0x146208: test_run_ (acutest.h:1178)
==29372==    by 0x14745B: main (acutest.h:1700)
==29372==  Block was alloc'd at
==29372==    at 0x4848899: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==29372==    by 0x59E64F: flb_malloc (flb_mem.h:80)
==29372==    by 0x5A29C7: out_lib_flush (out_lib.c:177)
==29372==    by 0x16D0B4: output_pre_cb_flush (flb_output.h:528)
==29372==    by 0xAD9B0A: co_init (amd64.c:117)
==29372== 

```